scripts/build.py

```diff
@@ -529,7 +560,7 @@ def should_file_be_built(filepath):
         bool. True if filepath should be built, else False.
     """
     if filepath.endswith('.js'):
-        return not any(
+        return not filepath in JS_FILEPATHS_NOT_TO_BUILD and not any(
```

Comment:
"filepath not in" is more pythonic, I think

Reply:
Done.